apps/comps/hooks/useUnlockKeys.ts

```diff
@@ -28,30 +78,12 @@ export const useUnlockKeys = (agentHandle: string, agentId?: string) => {
         // to get the agent ID relative to the sandbox (the sandbox vs production agent IDs differ)
         return await sandboxClient.getAgentApiKey(agentHandle);
       } catch (error) {
-        // If sandbox agent doesn't exist, create it. Note: this is a proactive action in case the
-        // "sync" between the main API and sandbox somehow gets out of sync
+        // If agent doesn't exist in sandbox yet, return null instead of failing
         if (
           error instanceof Error &&
           error.message.includes("Agent not found")
         ) {
-          const createResult = await sandboxClient.createAgent({
-            handle: agentHandle,
-            // Note: the name is technically required by the backend API, but all we need is the
-            // agent handle since this is globally unique and used for querying the API key
-            name: agentHandle,
-          });
-          if (!createResult.success) {
-            throw new Error("Failed to create sandbox agent");
-          }
-          const { id, apiKey, name } = createResult.agent;
-          return {
-            success: true,
-            agent: {
-              id,
-              apiKey,
-              name,
-            },
-          } as AdminAgentKeyResponse;
+          return null;
```

Comment:
and this was also the original behavior before https://github.com/recallnet/js-recall/pull/1224 landed